source/VersionHandler/src/VersionHandler.cs

```diff
@@ -165,7 +166,7 @@ private class FilenameComponents
             if (!string.IsNullOrEmpty(version)) {
                 int dotIndex = version.IndexOf('.');
                 if (dotIndex > 0 && version.Length > dotIndex + 1) {
-                    if (!float.TryParse(version.Substring(0, dotIndex + 2), out result)) {
+                    if (!float.TryParse(version.Substring(0, dotIndex + 2), NumberStyles.Any, CultureInfo.InvariantCulture, out result)) {
```

Comment:
So, `float.TryParse()` is locale-sensitive by default. Thus, `float.TryParse("5.5"; out result)` will return true on an English machine, but false on a German one.